install.sh

```diff
@@ -0,0 +1,2 @@
+./strato --compose
+./strato --pull
```

Comment:
Both steps can be skipped if using the release version. So may be can we get rid of install.sh entirely then?

Reply:
I'll remove that file